.gitignore

```diff
@@ -0,0 +1 @@
+examples/
```

Comment:
I find it a good idea to have some examples that the user can run, with an explanation how to do so in the README for instance.